tests/unit/pages/find-offers/FindOffers.spec.jsx

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from '@testing-library/react'
+import { vi } from 'vitest'
+import FindOffers from '~/pages/find-offers/FindOffers'
+import { ModalProvider } from '~/context/modal-context'
+
+const mockCloseModal = vi.fn()
+
+vi.mock('~/context/modal-context', () => ({
+  ModalProvider: ({ children }) => <div>{children}</div>,
+  useModalContext: () => ({
+    closeModal: mockCloseModal
+  })
+}))
+
+describe('FindOffers component', () => {
+  it('renders the Find Offers page', async () => {
+    render(
+      <ModalProvider>
+        <FindOffers />
+      </ModalProvider>
```

Comment:
Use `renderWithProviders` from '~tests/test-utils'

Reply:
@vlad-khrychov corrected the PR accordingly, unfortunately, I can't use the ~ custom src in this unit test

Reply:
@vlad-khrychov fixed